libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -763,6 +763,23 @@ void AC_PosControl::init_xy_controller()
     init_ekf_xy_reset();
 }
 
+/// standby_xyz_reset - resets I terms and removes position error
+///     This function will let Loiter and Alt Hold continue to operate
+///     in the event that the flight controller is in control of the
```

Comment:
should there be a "not" in this sentence?

Reply:
No, this is correct. We ensure the aircraft can still fly in standby to prevent a crash if it is mistakenly given control of the aircraft when in standby.